data/tests.yml

```diff
@@ -846,9 +846,10 @@
   skip_kotlin: true
   tags:
     highway: "road"
-    lanes: "3"
     sidewalk: "no"
     shoulder: "no"
+    lanes: "3"
+    lanes:forward: "2"
```

Comment:
can this not be inferred from `bus:lanes:forward`
what happens when this is removed?